test/functional/test_runner.py

```diff
@@ -152,6 +152,7 @@
     'p2p_node_network_limited.py',
     'feature_blocksdir.py',
     'feature_config_args.py',
+    'rpc_help.py',
     'feature_help.py',
```

Comment:
Imo this could be combined into the feature_help test. Just adding a new function there `def rpc_help(self): ...` and moving the other code into `def cli_help(self): ...`.